lib/simple_sqs/processor.rb

```diff
@@ -6,26 +6,26 @@
 require 'logger'
 
 class SimpleSqs::Processor
-  def process_sqs_message sqs_message
+  def process_sqs_message json_message_body, sqs_message = nil
```

Comment:
Is this method generally used directly by outside users of the lib? If not, I think you could make the second arg non-optional so it isn't accidentally left out.